.github/workflows/main.yml

```diff
@@ -19,7 +19,16 @@ jobs:
         run: |
           python -m pip install tox
       - name: Run tests
-        run: tox
+        run: tox -- --junit-xml=.test-results/pytest/cov.xml
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          file: .test-results/pytest/cov.xml  # Explicitly point at the correct coverage report
```

Comment:
This setting is documented as optional. Let's find out what the default setting is for the codecov uploader (I think it's `coverage.xml`) and make sure to produce the file using that filename and eliminate this optional setting.

Reply:
Fixed in 725be0142.